modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -317,8 +317,8 @@
 (defn- in-report-zone
   "Returns a HoneySQL form to interpret the `expr` (a temporal value) in the current report time zone, via Presto's
   `AT TIME ZONE` operator. See https://prestodb.io/docs/current/functions/datetime.html"
-  [expr]
-  (let [report-zone (qp.timezone/report-timezone-id-if-supported :presto-jdbc (lib.metadata/database (qp.store/metadata-provider)))
+  [driver expr]
+  (let [report-zone (qp.timezone/report-timezone-id-if-supported driver (lib.metadata/database (qp.store/metadata-provider)))
```

Comment:
unrelated but this is a big no-no because if anyone ever creates a new driver based on `:presto-jdbc` it will be instantly broken because their own methods for `qp.timezone/report-timezone-id-if-supported` would never get called. Imagine if Postgres did this, how would Redshift even work